circle.yml

```diff
@@ -8,39 +8,33 @@ jobs:
       CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
```

Comment:
Updates to this file are to allow support for python 3.6 (things like f-strings)